src/Renderer/DisplayDesc.cpp

```diff
@@ -13,4 +15,29 @@ namespace NAS2D
 		return !(a == b);
 	}
 
+
+	bool operator<(const DisplayDesc& a, const DisplayDesc& b)
+	{
+		const auto adims = std::make_pair(a.width, a.height);
+		const auto ap = std::make_pair(adims, a.refreshHz);
+		const auto bdims = std::make_pair(b.width, b.height);
+		const auto bp = std::make_pair(bdims, b.refreshHz);
+		return ap < bp;
```

Comment:
Might be easier to use [`std::make_tuple`](https://en.cppreference.com/w/cpp/utility/tuple/make_tuple) for something like this. Tuples support [comparison operators](https://en.cppreference.com/w/cpp/utility/tuple/operator_cmp) as well.